src/com/javacore/lesson3/SuperArrayList.java

```diff
@@ -0,0 +1,72 @@
+package com.javacore.lesson3;
+
+import java.util.Arrays;
+
+public class SuperArrayList<Type> implements BestList<Type> {
+
+    private static final int LIST_SIZE = 0;
+    private static Object[] array = new Object[LIST_SIZE];
+    private static Object[] temporaryArray;
+
+    @Override
+    public Type get(int index) {
+        return (Type) array[index];
```

Comment:
What if no element by index?
For example, you pass index 10 and array length 5. Need to add validation here.